apps/myaccount/src/helpers/user.ts

```diff
@@ -34,8 +33,12 @@ export const resolveUserDisplayName = (state: AuthStateInterface): string => {
     if (state.profileInfo.displayName) {
         return state.profileInfo.displayName;
     } else if (state.profileInfo.name.givenName || state.profileInfo.name.familyName) {
-        const givenName = isEmpty(state.profileInfo.name.givenName) ? "" : state.profileInfo.name.givenName + " ";
-        const familyName = isEmpty(state.profileInfo.name.familyName) ? "" : state.profileInfo.name.familyName;
+        const givenName: string = state?.profileInfo?.name?.givenName ?? "";
+        const familyName: string = state?.profileInfo?.name?.familyName ?? "";
+
+        if (givenName && familyName) {
+            return givenName + " " + familyName;
+        }
```

Comment:
Seems like the following code snippet is duplicated, better to move it to a seperate function.

If this is an urgent task, this is okay, else let's move this to a seperate function

Reply:
I will send a separate PR refactoring this.